guides/getting-started/set-up/openid4vc.md

```diff
@@ -0,0 +1,37 @@
+# OpenID for Verifiable Credentials
+
+The OpenID4VC module provides support for the [OpenID for Verifiable Credentials group of protocols](https://openid.net/sg/openid4vc/) defined under the OpenID Foundation. Currently this includes the [OpenID for Verifiable Credential Issuance](https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0.html), [Self-Issued OpenID Provider v2](https://openid.net/specs/openid-connect-self-issued-v2-1_0.html), and [OpenID for Verifiable Presentations](https://openid.net/specs/openid-4-verifiable-presentations-1_0.html).
+
+For the current supported versions for any of the OpenID4VC protocols, please refer to the [OpenID4VC Feature](../../features/openid4vc.md) page.
+
+The OpenID4VC Module in Credo currently exposes three modules, one for each role in the triangle trust: `OpenId4VcIssuerModule`, `OpenId4VcHolderModule`, and `OpenId4VcVerifierModule`. The issuer and verifier modules are expected to run in a cloud environment, as they require several endpoints to be exposed to the public internet. The holder module can run in a cloud environment or on a mobile device.
+
+### Installing OpenID4VC Module
+
+When using Credo with OpenID4VC you need to install the `@credo-ts/openid4vc` module:
+
+```console
+yarn add @credo-ts/openid4vc@^0.5.3
```

Comment:
```suggestion
yarn add @credo-ts/openid4vc@0.5.3
```

as it is experimental, right?

Reply:
Hmm yes but then you also need to put all other packages without ^ to avoid version inconsistencies